tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/RetryTimeoutTest.java

```diff
@@ -45,12 +49,21 @@
 
     private @Inject RetryTimeoutClient clientForRetryTimeout;
     
+    private TCKConfig config = TCKConfig.getConfig();
+    
     @Deployment
     public static WebArchive deploy() {
+
+        final ConfigAnnotationAsset config = new ConfigAnnotationAsset()
+            .setValue(RetryTimeoutClient.class,"serviceA", Timeout.class,getConfig().getTimeoutInStr(500))
+            .setValue(RetryTimeoutClient.class,"serviceWithoutRetryOn", Timeout.class,getConfig().getTimeoutInStr(500))
+            .setValue(RetryTimeoutClient.class,"serviceWithAbortOn", Timeout.class,getConfig().getTimeoutInStr(500));
+
         JavaArchive testJar = ShrinkWrap
                 .create(JavaArchive.class, "ftRetryTimeout.jar")
                 .addClasses(RetryTimeoutClient.class)
-                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
+            .addAsManifestResource(config, "microprofile-config.properties")
+            .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
```

Comment:
Looks like this will break formatting? :-)

Reply:
:anguished: